src/userDashboard/dataFormat/ConstantsUserDashboard.js

```diff
@@ -0,0 +1,53 @@
+const PERIOD_OPTIONS = {
+    WEEK: "Week",
+    MONTH: "Month",
+    YEAR: "Year",
+    YEARS: 'Years'
+};
+
+const ACTIVITY_TIME_FORMAT_OPTIONS = {
+    SECONDS: "Seconds",
+    MINUTES: "Minutes",
+    HOURS: "Hours"
+};
+
+const TOP_TABS = {
+  BAR_GRAPH: "Activity",
+  LINE_GRAPH: "Words"
+}
+
+//date format used for making sure dates are translated 
+//in the same way when working with the data
+//this is _not_ the date format used for formatting dates to be shown on the graphs or datepicker 
+const DATE_FORMAT = "yyyy-MM-dd"; 
+
+const BAR_GRAPH_KEYS = {

```

Comment:
why is this defined in this file?
it's only used in the UserBarGraph.js
why not have the constants at the beginning of that file then?

Reply:
the DATE_FROMAT is used in both (now with updated name) ReadingAndExercisesTimeDataFormat TranslatedWordsDataFormat, that's why it's defined in this file. 

In both files, there is something like this: 

    _dates.forEach(day => {
      var stringDate = format(day, DATE_FORMAT);
...._